documentation/style-guide.rst

```diff
@@ -103,11 +103,16 @@ Unix
 Use simple language
 ===================
 
-Avoid esoteric phrasing where possible.  Our audience is world-wide and may not
-be native English speakers.
-
-Don't use Latin abbreviations like "e.g." or "i.e." where English words will do,
-such as "for example" or "that is."
+The documentation has a world-wide audience of varying English proficiency.
+To make the documentation more accessible and inclusive, the following should be
+avoided:
+
+- Esoteric phrasing.
+- Abbreviations, such as Latin "e.g." or "i.e.," or symbols, such as the
+  ampersand ('&'), where English words will do, such as
+  "for example", "that is" and "and," respectively.
```

Comment:
I don't agree with discouraging '&'. It's relatively common, and helps distinguish items in a list, e.g. "Tom & Jerry and Barbie & Ken have both featured in major animated works".

Reply:
```suggestion
- Abbreviations, such as Latin "e.g." or "i.e." or symbols, such as the
  ampersand ("&"), where English words will do, such as
  "for example", "that is" and "and" respectively.
```

> I don't agree with discouraging '&'. It's relatively common

The same could be -- and was -- said for "e.g." and "i.e." too.
I wonder if this made more sense for printed media, since nowadays people can easily look those terms up and learn them instead.